spec/models/user_spec.rb

```diff
@@ -260,8 +271,8 @@
       before do
         create(:invoice_product, user_product: user_with_product.user_products.first)
         create(:invoice_product,
-          user_product: user_with_product.user_products.first,
-          invoice_settled: false)
+               user_product: user_with_product.user_products.first,
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.